### ISSUE TITLE
Algopool "trasferToMiner" function spelling

### ISSUE THREAD

**Issue by bearcanrun** [AdvancedAlgos/ALGOToken#7]
### Update spelling of "trasferToMiner" to "transferToMiner"

1. In Algopool.sol (labs/algo-token-distribution/src/AdvancedAlgos.AlgoToken.AlgoTokenDistribution/SmartContracts/src/AlgoPool.sol):

```
#line 29
function trasferToMiner(address minerAddress) public notTerminated onlyCoreTeam {

to
function transferToMiner(address minerAddress) public notTerminated onlyCoreTeam {
```

2. Update tests in AlgoPoolTests.cs (labs/algo-token-distribution/test/AdvancedAlgos.AlgoToken.AlgoTokenDistribution.IntegrationTests/AlgoPoolTests.cs)

**Comment by pmmax** [comment]
Contract function name, all related code, tests, playground and docs updated:
https://github.com/AdvancedAlgos/ALGOToken/pull/8

**Comment by bearcanrun** [comment]
Fixed by #8
